system/kube-monitoring/charts/prometheus-frontend/openstack-manila.alerts

```diff
@@ -0,0 +1,49 @@
+### Manila Alerts ###
+
+ALERT OpenstackManilaSharesStuck
+  IF (sum(openstack_manila_shares_stuck_count_gauge) by (host,status,check)) > 0
+  FOR 5m
+  LABELS {
+    service = "manila",
+    severity = "info",
+    tier = "openstack",
+    dashboard = "manila"
+    meta = "{{ $labels.check }}"
```

Comment:
I suggest `meta = "{{ $value }} shares"` to just give a count of the stuck shares as meta label